README.md

```diff
@@ -307,6 +307,40 @@ can then source these virtual environments using the instructions outlined
 a line such as this to each job destination:
 
     <env file="/path/to/shared/galaxy/venv" />
+    
+Using an external Grid Engine cluster
+-------------------------------------
+
+Almost things is as same as Slurm cluster.
+
+To use Grid Engine ( Sun Grid Engine, Open Grid Scheduler )
+ , one file and setup un environment variable are required:
+
+ 1. set environment SGE_ROOT
+ 2. create /var/lib/gridengine/default/common/act_qmaster file
```

Comment:
Is this one always called `act_qmaster`?
What do you think about adding this to the startup script, similar to the homepage stuff. If `act_qmaster` exists in /export/ symlink it to `/var/lib/gridengine/default/common/act_qmaster`. With this no manipulation of the container is required.

We could also mount this single file into the container?